cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -1599,7 +1599,7 @@ label:updated[]
 queryId
 ----
 a|
-The `queryId` procedure format has changed, and no longer includes the database name. For example, `mydb-query-123` is now `query-123`. This change affects built-in procedures `dbms.listQueries()`, `dbms.listActiveLocks(queryId)`, `dbms.killQueries(queryIds)` `and dbms.killQuery(queryId)`.
```

Comment:
This is from a list of things that changed in 4.1 - I'm not sure if we actually need to change this. Would be interesting to see what the docs team think.

Reply:
We should add it to the first section Version 5.0 ->  Removed features if not already added.

Reply:
makes sense, thanks